docs/contributing/contribution-checklists.md

```diff
@@ -626,8 +626,8 @@ guidelines.
    through long feedback cycles on a big PR with many resources. We ask you to
    only submit **1 resource at a time**.
 - [ ] __Acceptance tests__: New resources should include acceptance tests
-   covering their behavior. See [Writing Acceptance
-   Tests](#writing-acceptance-tests) below for a detailed guide on how to
+   covering their behavior. See [Writing An Acceptance
+   Test](running-and-writing-acceptance-tests.md#writing-an-acceptance-test) seciton for a detailed guide on how to
```

Comment:
There's a typo.

```suggestion
   Test](running-and-writing-acceptance-tests.md#writing-an-acceptance-test) section for a detailed guide on how to
```

Reply:
Thanks for your review. I have fixed it.